distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -0,0 +1,363 @@
+|/---------------+------------+------+------------+---------\|
+||cDEF           |cDEF               |cDEF        |cDEF     ||
+||  Transformers |                   | Federation | Sources ||
+|+---------------+ Catalog Framework +------------+---------+|
+||cDEF           |                   |cDEF   Eventing       ||
+||   Catalog     |                   +------------+---------+|
+||   Plugins     |                   |cDEF   Resources      ||
+|\---------------+-------------------+----------------------/|
+|                |cDEF               |                       |
+|                | Catalog Provider  |                       |
+|                +-------------------+                       |
+|                |cDEF               |                       |
+|                | Storage Provider  |                       |
+|                \-------------------/                       |
++------------------------------------------------------------+
+....
+
+The Catalog stores and translates Metadata, which can be transformed into many data formats, shared, and queried.
+The primary form of this metadata is the metacard. 
+A `Metacard` is a container for metadata. 
+`CatalogProviders` accept `Metacards` as input for ingest, and `Sources` search for metadata and return matching `Results` that include `Metacards`.
+
+=== Metacards
+
+A metacard is a single instance of metadata in the Catalog (an instance of a metacard type) which generally contains metadata providing a title for a product and describing that product's geo-location, created and modified dates, owner or producer, security classification, etc. 
+
+==== Metacard Type
+
+A metacard type indicates the attributes available for a particular metacard.
+It is a model used to define the attributes of a metacard, much like a schema.
+
+===== Default Metacard Type and Attributes
+
+Most metacards within the system are created using the default metacard type or a metacard type based on the default type.
+The default metacard type of the system can be programmatically retrieved by calling `ddf.catalog.data.BasicTypes.BASIC_METACARD`.
+The name of the default `MetacardType` can be retrieved from `ddf.catalog.data.MetacardType.DEFAULT_METACARD_TYPE_NAME`.
+
+The default metacard type has the following required attributes.
+Though the following attributes are required on all metacard types, setting their values is optional except for ID.
+
+.[[_metacard_required_attributes]]Metacard Required Attributes
+[cols="4*", options="header"]
+|===
+
+|`Metacard` Constant
+|Attribute Name
+|Attribute Format
+|Description
+
+|`CONTENT_TYPE`
+|`metadata-content-type`
+|STRING
+|Attribute name accessing for the metadata content type of a Metacard.
+
+|`CONTENT_TYPE_VERSION`
+|`metadata-content-type-version`
+|STRING
+|Attribute name for the version of the metadata content accessing type of a Metacard.
+
+|`CREATED`
+|`created`
+|DATE
+|Attribute name for the date/time *this Metacard* was created.
```

Comment:
I thought that the taxonomy actually uses `metacard.created` now to indicate the date the metacard was created, and that this is referencing the date the product was created. See https://codice.atlassian.net/wiki/display/DDF/metacard.created versus https://codice.atlassian.net/wiki/display/DDF/created